tuner/tuner/libtuner.py

```diff
@@ -236,20 +237,59 @@ class ParsedDisptachBenchmarkResult:
 class IREEBenchmarkResult:
     # Default format follows output of iree-benchmark-module
     candidate_id: int
-    result_str: str
+    result_json: list
 
     def get_mean_time(self) -> Optional[float]:
-        if not self.result_str:
-            return None
-        pattern = r"process_time/real_time_mean\s+([\d.]+)\s\w{2}"
-        match = re.search(pattern, self.result_str)
-        if not match:
-            return None
-        try:
-            return float(match.group(1))
-        except ValueError:
+        """Compute the mean time for all of the benchmarks - The result will be in ms"""
```

Comment:
Let's use microseconds. This unit is more realistic for the kind of dispatches we care about

```suggestion
        """Compute the mean time (in microseconds) for all of the benchmarks"""
```